### ISSUE TITLE
[💡SUG] Adding some recent non-sampling recommendation methods

### ISSUE THREAD

**Issue by chenchongthu** [RUCAIBox/RecBole#491]
Are you interested in adding some recent non-sampling recommendation methods like [ENMF](https://github.com/chenchongthu/ENMF)?  They are very effective but have not been widely noticed yet.

**Comment by ShanleiMu** [comment]
Thanks for your suggestion. We will consider implementing this kind of methods. :)

**Comment by deklanw** [comment]
@chenchongthu I'm interested in this!

So far in RecBole I've found LightGCN to give me the best results for general recommendation. From what I saw in the ENMF repo it looks like the performance is similar? You only give a comparison on one dataset again LightGCN, and none in the paper. 

I haven't tried content-aware yet but similarly ENSFM is claiming very promising results there.

The code for both would need to be ported from TF to PyTorch, yes?
